controllers/articles.controllers.js

```diff
@@ -0,0 +1,12 @@
+const {fetchArticleById} = require('../models/articles.models')
+
+function getArticleById(req, res, next){
+    
+    const {article_id} = req.params
+    return fetchArticleById(article_id).then((article) =>{
```

Comment:
It's not causing you any harm now but we don't have a reason to return this promise chain since it's not going to be used in a different context once we send the response.